### PR TITLE
Can resolve the classpath of the invisible project

### LINKED ISSUE #348: Single File Test Support
The plugin currently requires a maven/gradle project to run the unit tests which does not make sense. 
1. The debugger support single non project file run/debug.
2. Individual file test works smoothly for python, so I hope the java test runner can also run like that, even all output is just in console.

Is there a timeline for when single/individual file unit test running will be supported?

### PR DESCRIPTION
**Pull request by jdneo**
Resolve #348

### CODE REVIEW

File: java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/RuntimeClassPathUtils.java [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-* Copyright (c) 2018 Microsoft Corporation and others.
-* All rights reserved. This program and the accompanying materials
-* are made available under the terms of the Eclipse Public License v1.0
-* which accompanies this distribution, and is available at
-* http://www.eclipse.org/legal/epl-v10.html
-*
-* Contributors:
-*     Microsoft Corporation - initial API and implementation
-*******************************************************************************/
+ * Copyright (c) 2018 Microsoft Corporation and others.
```
Comment (yaohaizh) [inline java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/RuntimeClassPathUtils.java]: Update the time when add new changes.

Comment (jdneo) [reply java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/RuntimeClassPathUtils.java]: Updated

File: java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/RuntimeClassPathUtils.java [hunk 2]
```diff
@@ -49,9 +52,10 @@
         for (final IPath testPath : testPaths) {
             final Iterator<IJavaProject> iterator = javaProjectSet.iterator();
             while (iterator.hasNext()) {
-                final IJavaProject project = iterator.next();
-                if (project.getProject().getLocation().isPrefixOf(testPath)) {
-                    projectsToTest.add(project);
+                final IJavaProject javaProject = iterator.next();
+                final IProject project = javaProject.getProject();
+                if (belongToVisibleProject(testPath, project) || belongToInvisibleProject(testPath, project)) {
```
Comment (yaohaizh) [inline java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/RuntimeClassPathUtils.java]: Combine these to API. 

Just include in Workspace in okay

Comment (jdneo) [reply java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/RuntimeClassPathUtils.java]: Updated

**Pull request merged**